packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2148,18 +2148,14 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             }
         }
 
-        this.blobManager.updateUsedRoutes(blobManagerUsedRoutes);
```

Comment:
I think the reason why I had `updateUsedRoutes` here was to keep tombstone state consistent even when the tombstone feature was turned off. I don't think I have such a test, now that I think of it, do we need one? If we're trying to keep state consistent, I wonder if this is important enough to warrant that sort of change.

Reply:
Actually, I had added this to remove tomsbtone state if a tombstoned attachment blob is referenced. But, with this change, tombstone is separately handled. If tombstone is turned off, nothing will be set as tombstone in the first place, so this should not be needed.
The only reason this can be needed (in the future) is if BlobManager wants to do something with the used attachment blobs, which it currently does not.